src/Microsoft.AspNetCore.ResponseCompression/BrotliCompressionProviderOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO.Compression;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.ResponseCompression
+{
+    /// <summary>
+    /// Options for the <see cref="BrotliCompressionProvider"/>
```

Comment:
Relating to options, can you add a comment to ResponseCompressionOptions saying the Providers's priority is based on their ordering in the list?